sympy/ntheory/generate.py

```diff
@@ -302,12 +303,26 @@ def __contains__(self, n):
         a, b = self.search(n)
         return a == b
 
+    def __iter__(self):
+        for n in count(1):
+            yield self[n]
+
     def __getitem__(self, n):
-        """Return the nth prime number"""
+        """Return the nth (with n >= 1) prime number.
+
+        As sieve[0] does not exists, accessing it, even by using
+        sieve[:5] will raise an IndexError.
```

Comment:
That should be "exist" rather than "exists". I'm not sure that this change the docstring is beneficial though as no one will read the dicstring for `__getitem__` so this can be added as a comment.

Reply:
Oh thanks for proofreading my english, I hesitated on this one :] I reworked my docstring as advised.